<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SyvDaItChartDataItemWidget</class>
 <widget class="QWidget" name="C_SyvDaItChartDataItemWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>30</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>30</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>30</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,1,0">
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeChxDefaultSmall" name="pc_CheckBoxDataSerieActive">
     <property name="minimumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>22</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeLabColorOnly" name="pc_LabelColor">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pc_LabelIcon">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>15</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>15</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeLabDashboardChart" name="pc_LabelDataSerieName">
     <property name="text">
      <string>Signal A</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="stw_opensyde_gui_elements::C_OgeLabDashboardChart" name="pc_LabelValue">
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabColorOnly</class>
   <extends>QLabel</extends>
   <header>C_OgeLabColorOnly.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeChxDefaultSmall</class>
   <extends>QCheckBox</extends>
   <header>C_OgeChxDefaultSmall.h</header>
  </customwidget>
  <customwidget>
   <class>stw_opensyde_gui_elements::C_OgeLabDashboardChart</class>
   <extends>QLabel</extends>
   <header>C_OgeLabDashboardChart.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
